pkg/cache/sql/informer/indexer.go

```diff
@@ -192,7 +192,7 @@ func (i *Indexer) Index(indexName string, obj any) ([]any, error) {
 // ByIndex returns the stored objects whose set of indexed values
 // for the named index includes the given indexed value
 func (i *Indexer) ByIndex(indexName, indexedValue string) ([]any, error) {
-	rows, err := i.QueryForRows(context.TODO(), i.listByIndexStmt, indexName, indexedValue)
+	rows, err := i.QueryForRows(context.TODO(), listByIndexFmt, i.listByIndexStmt, indexName, indexedValue)
```

Comment:
This is the only part I don't get. The call to `i.QueryForRows` adds a second argument that's a template (or format), not the final query.  When I follow it into the implementation function `QueryForRows` at https://github.com/moio/lasso/blob/2bb0987964ad696e592689eef25ee4dba7b853f6/pkg/cache/sql/db/client.go#L115 it looks like the raw template is inserted into the `Error while running query` string, not the final query with the arguments injected into the template.

Am I missing something?